docs/data-sources/exporter.md

```diff
@@ -0,0 +1,24 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "timescale_exporter Data Source - terraform-provider-timescale"
+subcategory: ""
+description: |-
+  Exporter data source
+---
+
+# timescale_exporter (Data Source)
+
+Exporter data source
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of this exporter. Exporter names must be unique in order to manage them using Terraform.
```

Comment:
Since we can have exporters with the same name, this isn't right
There is no issue with having two exporters with the same name since they're managed (ie attached) using the ids

Reply:
there was an internal meeting where we decided to go forward with import by name, otherwise we'll have to start showing IDs in the UI.  This seems similar to have VPC works? It also fetches VPCs by name

Reply:
yes but names are unique for vpc by design

Reply:
is the solution you're thinking of to show IDs in the UI?